script.py

```diff
@@ -25,45 +25,45 @@ def generate_token(client_id, client_secret):
 
     if(token_response.status_code == 200):
         return token_response.json()['access_token']
-    else:
-        print('Invalid credentials')
 
 
-def request_vehicles(*arg):
-    if len(arg) == 1:
-        return requests.get(f'{osu_API}v1/beaverbus/vehicles', headers={
-            'Authorization': f'Bearer {arg[0]}'
-        })
-    if len(arg) == 2:
-        return requests.get(f'{osu_API}v1/beaverbus/vehicles/{arg[1]}',
-                            headers={'Authorization': f'Bearer {arg[0]}'}
+def request_vehicles(id):
+    authorization_header = {'Authorization': f'Bearer {token}'}
+    beaverbus_vehicles = 'v1/beaverbus/vehicles/'
+    if id:
```

Comment:
`id` is the reserved keyword in Python, can you rename to maybe `bus_id` or `vehicle_id`?